doc/en/user/source/styling/workshop/ysld/linestring.rst

```diff
@@ -485,11 +498,11 @@ Finished early? Here are some opportunities to explore what we have learned, and
 In a classroom setting please divide the challenges between teams (this allows us to work through all the material in the time available).
 
 .. only:: instructor
-  
-   .. admonition:: Instructor Notes 
+
+   .. admonition:: Instructor Notes
 
       As usual the Explore section invites readers to reapply the material covered in a slightly different context or dataset.
- 
+
       The use of selectors using the roads **type** attribute provides this opportunity.
```

Comment:
The geoserver template does not show instructor notes, need to run sphinx with -t instructor or something (so this can probably be removed)